### DISCUSSION TITLE
v0.21.4 (I assume?)

### DISCUSSION THREAD

**Discussion by bqcrabtree** [saber-notes/saber#1220] [General]
Highly excited for this, as it's the one that will include the change to let me use syncthing to sync my notes. Any idea when you plan to cut it? Got everything set up on all my devices (PC/laptop/tablet/phone) ready to go for the update.

So far this is the best note-taking software I've found for my tablet, so thank you for putting so much work into it. 💜

**Comment by adil192** [comment]
Sorry for the delay! v0.21.4 is up

**Reply by bqcrabtree** [reply]
Wonderful! Sadly, after exporting my notes and installing from here (since f-droid hasn't listed it yet,) I found that notes exported can't be imported as anything other than a blank canvas. That's okay though. It's only a few hours of doodles. Thank you for pushing this out. I really do love the app.
